packages/app/src/runner/event-manager.ts

```diff
@@ -452,6 +455,8 @@ export class EventManager {
   }
 
   _addListeners () {
+    addTelemetryListeners(this.getCypress)
```

Comment:
Maybe just pass `Cypress` directly?

```suggestion
    addTelemetryListeners(Cypress)
```

Reply:
I was following along with the 'handlePausing' event which uses getCypress.

[handlePausing](https://github.com/cypress-io/cypress/blob/matth/misc/telemetry/packages/app/src/runner/event-manager.ts#L626)

I can honestly say I don't know why it's getting cypress in this way and not just passing it directly.

Reply:
I switched to just Cypress... 🤷‍♂️